src/cli.js

```diff
@@ -83,11 +83,11 @@ const commandMap = {
   WORKSPACE_REMOVE: { remove: true, rm: true },
   WORKSPACE_RUN: { run: true },
   WORKSPACE_UPGRADE: { upgrade: true },
-  WORKSPACES: { workspace: true, w: true },
+  WORKSPACES: { workspaces: true, ws: true },
```

Comment:
The only change to this file was actually here, updated to say `workspaces` and `ws`, the rest is just prettier.
(do you have a custom prettier config btw?)